### PR TITLE
Add virtual IP entry

### PR DESCRIPTION
**Pull request by marian-pritsak**
Add entry to let overlay pipeline know which
destination IP to accept.

Signed-off-by: Marian Pritsak <marianp@mellanox.com>

### CODE REVIEW

File: experimental/saitypesextensions.h [hunk 1]
```diff
@@ -48,6 +48,8 @@ typedef enum _sai_object_type_extensions_t
 
     SAI_OBJECT_TYPE_ENI,
 
+    SAI_OBJECT_TYPE_VIP_ENTRY,
```
Comment (kcudnik) [inline experimental/saitypesextensions.h:51]: why adding in the middle and not at the end?

Comment (marian-pritsak) [reply experimental/saitypesextensions.h:51]: It was autogenerated
Is it required to keep these enums in order?

**Comment by prsunny** [comment]
@kcudnik , I just noticed one outstanding comment . Are you ok that this got merged?

**Comment by kcudnik** [comment]
> @kcudnik , I just noticed one outstanding comment . Are you ok that this got merged?

should be good enough, but i would prefer this to be in order, since currently extensions are not checked for that, but if we start using them in warm boot then there will be an issue with this

**Pull request merged**